src/common/SPELLS/TALENTS/SHAMAN.js

```diff
@@ -43,6 +43,7 @@ export default {
   HIGH_VOLTAGE_TALENT: { id: 260890, name: "Unlimited Power", icon: "ability_vehicle_electrocharge"},
   MASTER_OF_THE_ELEMENTS_TALENT: { id: 16166, name: "Master of the Elements", icon: "spell_nature_elementalabsorption"},
   EXPOSED_ELEMENTS_TALENT: { id: 260694, name: "Exposed Elements", icon: "spell_nature_callstorm"},
+  EXPOSED_ELEMENTS_DEBUFF: { id: 269808, name: "Exposed Elements Debuff", icon: "spell_nature_callstorm"},
```

Comment:
Please add the debuff id in `/SPELLS/SHAMAN` instead, otherwise it will be removed once we auto generate the talent files. (The auto generate only generates the ID of the talents, not any related debuffs or buffs).